docs/known_issues.rst

```diff
@@ -75,6 +75,50 @@ Both will throw an exception if units do not cancel, e.g.::
 
 See: https://github.com/astropy/astropy/issues/7582
 
+Multiplying a `pandas.Series` with an `~astropy.units.Unit` does not produce a |Quantity|
+-----------------------------------------------------------------------------------------
+
+Quantities may work with certain operations on `~pandas.Series` but
+this behaviour is not tested.
+For example, multiplying a `~pandas.Series` instance
+with a unit will *not* return a |Quantity|. It will return a `~pandas.Series`
+object without any unit::
+
+.. doctest-requires:: pandas>=1.5
```

Comment:
The `doctest-requires` instructions are appearing in the rendered documentation.
```suggestion
object without any unit:

.. doctest-requires:: pandas>=1.5
```
I'm only highlighting it here, but the others need to be edited too.